pkgs/development/haskell-modules/configuration-nix.nix

```diff
@@ -1479,10 +1479,13 @@ self: super: builtins.intersectAttrs super {
     mpi-hs-binary
     ;
 
-  postgresql-libpq = overrideCabal (drv: {
-    # Using use-pkg-config flag, because pg_config won't work when cross-compiling.
-    configureFlags = drv.configureFlags or [] ++ [ "-fuse-pkg-config" ];
-  }) super.postgresql-libpq;
+  postgresql-libpq = lib.pipe super.postgresql-libpq [
```

Comment:
```suggestion
  postgresql-libpq = lib.pipe (super.postgresql-libpq.override {
    postgresql-libpq-configure = null;
   }) [
```

Reply:
Oh hadn't noticed this was declared as a dependency even though -pkgconfig wasn't.